tf2_gnn/models/graph_task_model.py

```diff
@@ -290,11 +290,17 @@ def _apply_gradients(
         clip_global_norm_val = self._params.get("gradient_clip_global_norm")
 
         if clip_val is not None:
+            if clip_norm_val is not None:
+                raise ValueError("Both 'gradient_clip_value' and 'gradient_clip_norm' are set, but can only use one at a time.")
+            if clip_global_norm_val is not None:
+                raise ValueError("Both 'gradient_clip_value' and 'gradient_clip_global_norm' are set, but can only use one at a time.")
             gradient_variable_pairs = [
                 (tf.clip_by_value(grad, -clip_val, clip_val), var)
                 for (grad, var) in gradient_variable_pairs
             ]
         elif clip_norm_val is not None:
+            if clip_global_norm_val is not None:
+                raise ValueError("Both 'gradient_clip_norm' and 'gradient_clip_global_norm' are set, but can only use one at a time.")
             gradient_variable_pairs = [
                 (tf.clip_by_norm(grad, clip_norm_val), var)
                 for (grad, var) in gradient_variable_pairs
```

Comment:
This looks a little verbose, I was thinking of something like
```python
if [clip_val, clip_norm_val, clip_global_norm_val].count(None) < 2:
    ...
```
but I guess this is also fine (easier to get pairwise error messages).